src/main/host/process.rs

```diff
@@ -1074,6 +1152,28 @@ impl Process {
                 .unwrap();
         }
 
+        // Initialize emulate rlimits to their native values.
+        // TODO: For determinism, we may want to use fixed limits for some or all of these.
+        // Some applications like Tor will change behavior depending on these limits.
+        // Ultimately they'd need to be compatible with the native limits though.
+        let mut rlimits: [linux_api::resource::rlimit64;
+            linux_api::resource::RLIM_NLIMITS as usize] =
```

Comment:
I think that this size `linux_api::resource::RLIM_NLIMITS as usize` could just be an underscore, rather than specifying it twice?